### ISSUE TITLE
Inline output of the chunk shows only every second time when run [MacOS ARM]

### ISSUE THREAD

**Issue by danieltomasz** [rstudio/rstudio#12286]
### System details

    RStudio Edition : Desktop
    RStudio Version : 2022.12.0-preview+311 
    OS Version      : MacOS 13.1 beta
    R Version       :  4.2.2 (2022-10-31)
   Platform: aarch64-apple-darwin20 (64-bit)

### Steps to reproduce the problem

Have an rmarkdown file (this happens also for quarto)
````
---
title: "R Notebook"
output: html_notebook
---

```{r}
print(2)
```
````

### Describe the problem in detail
The output stays visible only every second time when I run the chunk.
For the other times, the output is visible a short moment but it diseappers.

### Describe the behavior you expected

Output stays visible the whole time.

- [x] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [x] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [x] If I am reporting an RStudio crash, I have included a [diagnostics report](https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by danieltomasz** [comment]
The same behaviour I can observe for 2022.07.2 Build 576

**Comment by ronblum** [comment]
@danieltomasz Thank you for raising this! Do you know if this was a problem before upgrading to MacOS 13.1 beta or R 4.2.2?

**Comment by danieltomasz** [comment]
I don't remember such behaviour, I tried to downgrade to R 4.2.1 and behaviour is happening too, I dont want to downgrade MacOS at this stage

Another related behavior was that the if there was error when executing the chunk code, the other chunks didnt want to run after it, as it would be still executing ; after several reinstalls of Rstudio and R, I was able to fix it

**Comment by ronblum** [comment]
I'm currently using MacOS 13 (not 13.1), but I can't reproduce the problem.

@jonvanausdeln Though I doubt this would make a difference, can you check whether or not it's an issue on the M1?

**Comment by danieltomasz** [comment]
@ronblum  thank for following this issue.

I create a new test used profile on mac and there I run rmarkdown file without any problems.

So the problem was  Rstudio settings on my current user profile, after I deleted all the settings RStudio started working again. Would be useful to share with you the copy of the previous settings?
The behaviour improved after deleting the `~/.config/rstudio/rstudio-prefs.json` but I also deleted the `.local/share/rstudio`

[rstudio-prefs.json.zip](https://github.com/rstudio/rstudio/files/9938478/rstudio-prefs.json.zip)


Previously, also saving rmarkdown cleared all outputs of the chunks (if uthis ring a bell, maybe this a known issue, specific setting that I toggle not being aware of).

For me issue is closed, but you might want to investigate furthr
